### ISSUE TITLE
why my cv score is negative number?

### ISSUE THREAD

**Issue by iWangLin** [EpistasisLab/tpot#681]
my code 
from tpot import TPOTRegressor
from sklearn.datasets import load_boston
from sklearn.model_selection import train_test_split

housing = load_boston()
X_train, X_test, y_train, y_test = train_test_split(housing.data, housing.target,
                                                    train_size=0.75, test_size=0.25)

tpot = TPOTRegressor(generations=5, population_size=50, verbosity=2)
tpot.fit(X_train, y_train)
print(tpot.score(X_test, y_test))
tpot.export('tpot_boston_pipeline.py')

but the cv score is negative number as this:
![image](https://user-images.githubusercontent.com/34855146/38498752-04804a82-3c38-11e8-8307-e234cdb02ce4.png)
I really want to know why?
Thank You!

**Comment by weixuanfu** [comment]
Please check this same issue #595

**Comment by weixuanfu** [comment]
Also #675

**Comment by weixuanfu** [comment]
I close this issue. Please feel free to re-open the issue if you have any more questions

**Comment by iWangLin** [comment]
Thank you. I checked the two same issues, but I have not clearly known about the Tpot Regressor. The value of cv socre should be  negative ? Could you tell me where I can find the cv score of Tpot Regressor ？

**Comment by weixuanfu** [comment]
Related issue #425 #612

Please check TPOT API, the default scoring function is accuracy for TPOTClassifier, which is in range from 0 to 1. But it is neg_mean_squared_error for TPOTRegressor that stands for negated value of mean squared error.

**Comment by iWangLin** [comment]
Oh, I see. Thank you very much !
